docs/consent/README.md

```diff
@@ -0,0 +1,38 @@
+
+We created this module to help you readily obtain consent from your website users for various cookies that will be served on your website. With the features available in this module, a website's visitor will be able to control the _types_ of data or cookies that can be stored and shared if you choose to offer that level of control.
+
+## Why should I use it?
+To keep up with the GDPR regulations and your company's privacy and cookie policies. GDPR regulations mandate that businesses need user consent before they can collect or track any visitor data. This tool helps you create a cookie consent banner out of the box. With the banner options, a user can choose to select the cookies (data) that can be collected by the website. The module lets you manage first party and third party cookies.
+
+## How does it work?
+
+Altis Consent is an API that is designed to handle how cookies and other types of data stored locally on a user's computer are registered in the code. By creating a registry of cookies and local storage, and defining what those cookies are used for, the API allows the software to intelligently use only those cookies that a user has granted access to.
+
+In order for this to work, all cookies and local storage used on a site must be registered in the way mentioned in the Altis developer documentation.
+
+## What does it do?
+
+Out of the box, the Consent module handles all first-party cookies and local storage data. Using the controls provided on the Privacy page in the WP admin you can link your website's Privacy Policy and Cookie storage Policy page. There are options to control whether to grant the user a choice to select the types of cookies they want to consent to, or an option to allow all cookies or only functional cookies. You may also easily add a cookie consent banner message in the admin settings.
+
+In addition, a robust templating system and dozens of filters allow development teams to fully customize the options or the display of the banner, or only customize certain specific elements, based on a site's needs.
+
+## What about third-party cookies?
+The API allows management of third party cookies if it is configured so. Refer Developer Docs to see how this can be done.
+
+## Can I manage user data tracked through Altis' personalization features?
+Altis' personalization features track user data that comprise first-party cookies and those will be treated as explained [above](https://github.com/humanmade/altis-consent/wiki#what-does-it-do).
+
+## Does the API help categorize the type of cookies?
+Out of the box, [five categories](https://github.com/humanmade/altis-consent/wiki/WP-Consent-API#consent-categories) are provided functional, marketing, statistics, statistics-anonymous and preferences. More categories can be added via a filter as explained in the Dev Docs.
```

Comment:
May want to update this link if we have this info within these docs. It's fine to link out to the wiki still, just needs to be kept up to date.

Reply:
I think all of these links are simply because the docs themselves were cloned from the consent wiki's repo.